design/src/rfcs/rfc0016_body_callback_apis_v2.md

```diff
@@ -0,0 +1,389 @@
+RFC: Replacing Callback APIs for `ByteStream` and `SdkBody` with a body-wrapping API
+=================================================
+
+> Status: RFC
+
+Adding a body-wrapping API to `ByteStream` and `SdkBody` will enable developers using the SDK to implement things like checksum validations and 'read progress' callbacks. This RFC supersedes [RFC0013] and will partially replace the changes made for that RFC.
```

Comment:
Seems like we should update the status line in RFC 13 to say something like "Superceded by X"